<launch>

    <node name="blinker_detection_node" pkg="blinker_tracking" type="blinker_detection_node" output="screen">
        <remap from="~image_raw" to="/sync/cam0/image_raw"/>
        <param name="minThreshold" value="128" />
        <param name="maxThreshold" value="255" />       
        <param name="filterByColor" value="true" />
        <param name="filterByArea" value="true" />
        <param name="maxArea" value="200" />
        <param name="minArea" value="15" />
        <param name="filterByCircularity" value="false" />
        <param name="minCircularity" value="0.5" />
        <param name="filterByConvexity" value="false" />
        <param name="minConvexity" value="0.4" />
        <param name="filterByInertia" value="true" />
        <param name="minInertiaRatio" value="0.3" />
        <param name="minDistBetweenBlobs" value="30" />
    </node>

</launch>
